blog_article/blog_article.adoc

```diff
@@ -0,0 +1,161 @@
+= (WIP) AMQP Interoperability using RabbitMQ
+
+System architectures usually evolve over time: services and components are added, replaced or modified.
+For event-driven architectures, this means that events need to be dynamically routed and received by different
+applications. As part of this overlying scenario, we want to have a look at interoperability between the protocols
+AMQP 0.9.1 and AMQP 1.0. Because even if the names of the protocols are very similar, we will see that a few
+things must be taken into account in application development to be able to achieve interoperability.
+
+== AMQP 0.9.1 and AMQP 1.0 - The same Protocol?
+
+At first, one could assume that the protocols AMQP 0.9.1 and AMQP 1.0 are very similar and that the differences
+between them cannot be that severe. Unfortunately, the naming could lead to confusion for those who are not familiar
+with the concepts of both protocols. In fact, AMQP 0.9.1 and AMQP 1.0 should be treated like completely different
+specifications. The Advanced Message Queuing Protocol (AMQP) was released for communication between clients and
+messaging brokers. It is widely used and supported by several broker implementations: RabbitMQ, ActiveMQ and SwiftMQ,
+to name just a few examples. The architecture that is assumed by AMQP 0.9.1 relies mainly on queues and exchanges. +
+In contrast to that, AMQP 1.0 was developed to be more universally applicable and therefore does not know concepts
+like these entities. By assuming less, the protocol should be more flexible to use. +
+As also stated on the AMQP website, the protocol was introduced "to become the standard protocol for
+interoperability between all messaging middleware". Using the RabbitMQ broker as an example, we want to take
+a closer look at the extent to which AMQP is supported and how different versions interoperate with each other.
+Therefore, in the following, we will compare the use of three Java client libraries: RabbitMQ, SwiftMQ and Qpid JMS.
+The RabbitMQ broker requires an activated AMQP 1.0 plugin for this. +
+The competing consumer and publish-subscribe patterns were implemented using these clients within this
+https://github.com/NovatecConsulting/showcase-rabbitmq/tree/main[repository]. Feel free to have a look at the implementations
```

Comment:
I would also mention that it also contains implementations which show how interoperability could be achieved. I think you can also add a link to a specific implementation when you talk about it.